abi/auto_marshal_abi_spec_test.go

```diff
@@ -0,0 +1,102 @@
+// Copyright (C) 2024, Ava Labs, Inc. All rights reserved.
+// See the file LICENSE for licensing terms.
+
+package abi
+
+import (
+	"encoding/hex"
+	"encoding/json"
+	"os"
+	"strings"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+
+	_ "embed"
```

Comment:
Why is this imported? Are there side effects of this import in its `init` we depend on?

Reply:
Thanks for catching it! Old code